fabric-networking-api-v1/src/main/resources/fabric-networking-api-v1.accesswidener

```diff
@@ -1,2 +1,4 @@
 accessWidener v2 named
 
+accessible field net/minecraft/server/network/ServerLoginNetworkHandler transferred Z
+accessible field net/minecraft/server/network/ServerCommonNetworkHandler transferred Z
```

Comment:
```suggestion
transitive-accessible field net/minecraft/server/network/ServerLoginNetworkHandler transferred Z
transitive-accessible field net/minecraft/server/network/ServerCommonNetworkHandler transferred Z
```

Reply:
Shouldn't this be in the `transitive-access-wideners` module for performance reasons?